fix(curriculum): remove media query requirement, general improvements (#46639)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3485f0b20591d26d2a1.md

# --description--

> Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property with `10px` set as the value.

Reviewer: ```suggestion
Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property of `10px`.
```

Author: End of sentence code block.

Reviewer: I'm fine with your original change here.

> Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property with `10px` set as the value.

Reviewer 2: What do you think of?
```suggestion
To smooth out your images. Give `border-radius` property with a value of `10px` to `.gallery img` selector.
```

Author: The first part of the sentence sounds incomplete to me. I also don't really see an issue with the sentence as it is.

Reviewer 2: Ya this one is weird 😅

body {
  margin: 0;
  font-family: sans-serif;
  background: #f5f6f7;
}

.header {
  text-align: center;
  text-transform: uppercase;
  padding: 32px;
  background-color: #0a0a23;
  color: #fff;
  border-bottom: 4px solid #fdb347;
}

.gallery {
  display: flex;

>   flex-direction: row;

Reviewer 2: ```suggestion
  flex-direction: row-reverse;
```
